### ISSUE TITLE
Add contribuition guide: How to open pull request 

### ISSUE THREAD

**Issue by nynaalekhya** [backdrop/backdrop-issues#5124]
Proposed Section:
There is should be a beginners guide section which contributors can follow on how to open pull request : fork, clone, commit and push. And then wait for the feedback to approve the request.

**Comment by indigoxela** [comment]
We do have something in our official documentation: https://docs.backdropcms.org/documentation/contributors-guide.

But maybe it's not as easy to find as it should be. And, of course, documentation can always get improved.

@nynaalekhya do you have specific recommendations for improvement?

**Comment by nynaalekhya** [comment]
@indigoxela 
It would be good to include a collapsible section in the readme file on how to submit a PR as shown below 


<details>
  <summary>How to begin contributing</summary>
  
1. Fork the [repository](https://github.com/backdrop/backdrop.git).

2. Clone the forked repository and configure the upstream remote:

```sh
git clone https://github.com/<your username>/backdrop.git
cd backdrop
git remote add upstream https://github.com/backdrop/backdrop.git
```

3. Fetch the updates from the upstream branch and synchronize with the local `1.x` branch:

```sh
git fetch upstream
git checkout 1.x
git merge upstream/1.x
```
4. Create a new branch:

```sh
git checkout -b my-new-branch
```

5. Make changes to the code, then commit the changes and push to the forked repo:

```sh
git add .
git commit -m "This is my first commit"
git push -u origin my-new-branch
```

6. Go to the [repository](https://github.com/backdrop/backdrop.git) and link a Pull Request to the issue.

After reviewing the pull request, we will provide you with feedback, request further changes and approve the PR to merge the work.

</details>

**Comment by BWPanda** [comment]
I don't agree with putting this much documentation information in the README. This is why we have a Docs site. If the Contributors guide isn't obvious to find, we should address that. The most I'd recommend is a link to it from the README...

**Comment by indigoxela** [comment]
To be honest, I share @BWPanda's concern. Having the same docs in different places also means, that if one of those changes, we have to take care of the other one, too, which - worst case - might be forgotten.

But in my opinion a link in the README to the official docs is a very good idea. Let's make it as easy to find as possible.
People who directly come here might not yet know, where to search for the docs.

Note that there's a [related issue](https://github.com/backdrop/backdrop-issues/issues/3631) regarding adding a CONTRIBUTING.md to core.

**Comment by nynaalekhya** [comment]
It would be a good idea to add the existing documentation link to the readme file for easy access. This link https://docs.backdropcms.org/documentation/contribute-to-backdrop-core regarding how to contribute backdrop core could be included or do you'll recommend something else.

**Comment by indigoxela** [comment]
@nynaalekhya yes, that's the link I'd recommend to include in the README.md :+1:

**Comment by nynaalekhya** [comment]
I have updated the file, could you review the PR

**Comment by indigoxela** [comment]
> I have updated the file, could you review the PR

Cool, many thanks @nynaalekhya!
One thought: currently the link is below section "Thanks to the following companies...", which seems a bit disconnected to the pull request topic above.
Personally, I'd recommend to move it up a bit, closer to "When submitting a pull request...".

@BWPanda what do you think?

**Comment by BWPanda** [comment]
@indigoxela I don't see that... New text is with rest of PR text.

In any case, perhaps the text "When submitting a pull request" could become the link. Nice and simple.

**Comment by indigoxela** [comment]
@BWPanda oops, you're right, I might have reviews a wrong commit (?). :confused: 

Hm, I don't think that I'd assume the text "When submitting a pull request" to be a link to the documentation. But I might be wrong.

@nynaalekhya in your PR comment on https://github.com/backdrop/backdrop/pull/3655 please replace `Closes #5124` with `Fixes https://github.com/backdrop/backdrop-issues/issues/5124` (it has to be the complete link). Then your PR gets automatically linked here in the issue. Otherwise people (testers / reviewers) might have trouble to find it. (Like I did.)

**Comment by nynaalekhya** [comment]
@backdrop-ci has mentioned the Related to: [backdrop/backdrop-issues#5124](https://github.com/backdrop/backdrop-issues/issues/5124)
Sure, I'll add it to the PR description

**Comment by klonos** [comment]
Might wanna link to https://backdropcms.org/news/creating-pull-requests-against-core-easy-noob-way from somewhere - if people find it useful.

**Comment by bugfolder** [comment]
Agree with @klonos on the value of linking to that guide, though I think the appropriate place to put the link would be on the [Contribute to Backdrop core](https://docs.backdropcms.org/documentation/contribute-to-backdrop-core) page (I'll add an edit there), so that only one link is needed in the README.md file.

Code looks good to me. (An easy one, since it's documentation-only.)

**Comment by bugfolder** [comment]
Oops, I can't edit [Contribute to Backdrop core](https://docs.backdropcms.org/documentation/contribute-to-backdrop-core); insufficient permissions.

**Comment by bugfolder** [comment]
Link to noob guide to PRs added to the top of [Contribute to Backdrop core](https://docs.backdropcms.org/documentation/contribute-to-backdrop-core). Also a bit of rewording of the top-of-page text. (See revisions to see what changed.)

**Comment by stpaultim** [comment]
Good work on this issue. This seems like a good idea!

**Comment by BWPanda** [comment]
I requested some small changes to the PR...
